OpenOversight/app/models/database.py

```diff
@@ -102,18 +107,23 @@ def total_documented_assignments(self):
             .count()
         )
 
-    @cached(cache=DB_CACHE, key=db_model_cache_key(KEY_DEPT_TOTAL_INCIDENTS))
+    @cached(cache=DB_CACHE, key=model_cache_key(KEY_DEPT_TOTAL_INCIDENTS))
     def total_documented_incidents(self):
         return (
             db.session.query(Incident).filter(Incident.department_id == self.id).count()
         )
 
-    @cached(cache=DB_CACHE, key=db_model_cache_key(KEY_DEPT_TOTAL_OFFICERS))
+    @cached(cache=DB_CACHE, key=model_cache_key(KEY_DEPT_TOTAL_OFFICERS))
     def total_documented_officers(self):
         return (
             db.session.query(Officer).filter(Officer.department_id == self.id).count()
         )
 
+    @staticmethod
+    def remove_database_cache_entry(dept_id: int, update_type: List[str]) -> None:
+        """Remove the Department model key from the cache if it exists."""
+        remove_database_cache_entry(Department(id=dept_id), update_type)
```

Comment:
I am not really understanding why this is a static method. Why not
```suggestion
    def remove_database_cache_entry(self, update_types: List[str]) -> None:
        """Remove the Department model key from the cache if it exists."""
        remove_database_cache_entry(self, update_types)
```
Would this require additional database lookups?

Reply:
I don't believe so, no! Tbh, I started with a static method and it didn't quite change from there. Good call.